alacritty_terminal/src/event_loop.rs

```diff
@@ -82,6 +85,14 @@ impl event::Notify for Notifier {
     }
 }
 
+impl event::OnResize for Notifier {
+    fn on_resize(&mut self, size: &SizeInfo) {
+        if self.0.send(Msg::Resize(*size)).is_err() {
+            panic!("expected send event loop msg");
+        }
```

Comment:
I'd assume you've mirrored the other code in this file with this, right?

Generally a great idea, but I do not understand the purpose of doing this. Couldn't we just `expect("error")`? I'd assume the original author must have forgotten about it or something got confused in a refactoring.

If you're down, feel free to change all instances of this oddness in this file.

Reply:
Will do 👍